### PR TITLE
remove generator dependency in create service account

### PR DESCRIPTION
**Pull request by ramnar**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Removes generator dependency in create service account
**Which issue(s) this PR fixes**:

Part of #93100

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by rikatz** [comment]
/cc @rikatz

I would like to take a look :)

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_serviceaccount.go
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/create/create_serviceaccount.go:30]: `resource` is sufficient and widely used alias

**Review by soltysh — APPROVED** [review]
/triage accepted
/priority backlog
/lgtm
/approve
/ok-to-test

**Comment by rikatz** [comment]
I've seen a thread that there's some issue going on with git :/ Will wait until this gets fixed

**Comment by mrbobbytables** [comment]
git ref issue: https://github.com/kubernetes/test-infra/issues/19910

**Comment by rikatz** [comment]
/test pull-kubernetes-node-e2e-podutil
:crossed_fingers:

**Comment by rikatz** [comment]
/hold

Let's wait until the gh problem gets solved

I'll also propose to migrate those three failing jobs off of bootstrap.py so they can pass :)

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_serviceaccount.go
Comment (ramnar) [reply staging/src/k8s.io/kubectl/pkg/cmd/create/create_serviceaccount.go]: @soltysh  - done this change

**Comment by rikatz** [comment]
@ramnar can you please squash your commits? 

Then I can lgtm here

**Comment by ramnar** [comment]
@rikatz - I will squash merge. I will need some time as I am different challenges on different front which I am sorting out

**Comment by brianpursley** [comment]
/ping @ramnar looks like maybe all that is needed here is to squash the commits, just following up on that so your work doesn't go to waste

**Comment by rikatz** [comment]
@brianpursley because squashing is not actually required, I'm thinking about lgtm and remove the hold, and let this get merged with two commits so we wont lose the work.

WDYT?

**Comment by ramnar** [comment]
Please proceed

**Comment by rikatz** [comment]
@ramnar are you able to do the squash of commits?

**Comment by ramnar** [comment]
@rikatz  I have some operational issues which will take more time to resolve because of which I am unable submit this pull request.Apologies for this. I have to close this pull request. But any one else can feel free to use this code to submit new pull request.

**Comment by rikatz** [comment]
/reopen

Let's keep it open and will take care on monday :)

**Pull request merged**
